apps/storybook/stories/public/orama-chat-box.stories.tsx

```diff
@@ -72,7 +72,11 @@ const Template = ({ preset }) => {
       sourceBaseUrl=${preset?.sourceBaseUrl}
       .sourcesMap=${preset?.sourcesMap}
       .suggestions=${preset?.suggestions}
-    ></orama-chat-box>
+    >
+    <div slot="empty-state" style="display: flex; justify-content: center; align-items: center; padding: 1rem;">
+      <orama-text as="p">Optional empty content here...</orama-text>
+    </div>
```

Comment:
can you please remove it from the orama-chat-box story? the "empty-state" slot usage will be documented in https://docs.orama.com if needed.

Reply:
removed